Sources/Defaults/Defaults.swift

```diff
@@ -41,15 +41,24 @@ public enum Defaults {
 	}
 
 	public final class Key<Value: Serializable>: AnyKey {
-		public let defaultValue: Value
+		public typealias DefaultGetter = () -> Value
+		/**
+		The `defaultGetter` will be executed in these situations:
+
+		- `UserDefaults.object(forKey: string)` return `nil`
+		- `bridge` cannot deserialize `Value` from `UserDefaults`
+		 */
+		private let defaultGetter: DefaultGetter
+		public var defaultValue: Value { defaultGetter() }
 
 		/**
 		Create a defaults key.
 
 		The `default` parameter can be left out if the `Value` type is an optional.
 		*/
-		public init(_ key: String, default defaultValue: Value, suite: UserDefaults = .standard) {
-			self.defaultValue = defaultValue
+		public init(_ key: String, default defaultGetter: @autoclosure @escaping DefaultGetter, suite: UserDefaults = .standard) {
```

Comment:
I don't think it should be an autoclosure. It will mean that people can accidentally pass a function. I think this parameter should remain unchanged and just wrapped it in a closure internally.